Pat.Subscriber/AzureServiceBusMessageReceiverFactory.cs

```diff
@@ -15,5 +16,12 @@ protected override IMessageReceiver CreateMessageReceiver(string connectionStrin
             return new MessageReceiver(connectionString,
                 EntityNameHelper.FormatSubscriptionPath(topicName, subscriberName));
         }
```

Comment:
New line between these please, let's keep it neat :)

Reply:
new line where? between the params?